bcfms/src/bcfms/ipa/schema/InitialProjectReviewSchema.ts

```diff
@@ -0,0 +1,80 @@
+import { z } from 'zod';
+
+const InitialProjectReviewSchema = z.object({
+    dateSubmitted: z.date({
+        // This handles null values -> null !== typeof Date
+        invalid_type_error: 'Date Submitted is required.',
+    }),
+    assessmentDueDate: z.date(),
+    intersectsIFA: z.boolean().default(false),
+    proximityToFossils: z
+        .string({
+            invalid_type_error: 'Proximity To Fossils is required.',
+        })
+        .min(1, { message: 'Proximity To Fossils is required.' })
+        .max(250)
+        .nullable(),
+    groundDisturbance: z
+        .string({
+            invalid_type_error: 'Ground Disturbance is required.',
+        })
+        .min(1, { message: 'Ground Disturbance is required.' })
+        .max(250)
+        .nullable(),
+    metamorphicRock: z.string().max(250).nullable(),
```

Comment:
All the assessment criteria are concept datatypes (eg UUIDs). Please check in the resource models and align the datatypes with those.